packages/navigator/src/browser/navigator-model.ts

```diff
@@ -43,6 +43,11 @@ export class FileNavigatorModel extends FileTreeModel {
         this.initializeRoot();
     }
 
+    protected scrollToAlignment: 'auto' | 'center';
+    get scrollAlignment(): 'auto' | 'center' {
+        return this.scrollToAlignment;
+    }
```

Comment:
I'm not sure about these changes. The values `'auto'` and `'center'` are props from `TreeWidget.ViewProps` which likely should not leak to the model itself.